build.gradle

```diff
@@ -82,8 +82,11 @@ repositories {
 dependencies {
 
   compile group: 'cascading', name: 'cascading-hadoop2-mr1', version: cascadingVersion, changing: true
+  compile group: 'org.apache.commons', name: 'commons-csv', version: '1.2'
```

Comment:
Is it worth dragging in a completely new library for this or can we maybe get away with string operations.